web/src/utils/url.ts

```diff
@@ -43,29 +41,29 @@ export const getURLParamAsNumber = (arg: URLParam) => {
 
 export const setURLParams = (params: string) => {
   const url = new URL(window.location.href);
-  history.push(`${url.pathname}?${params}${url.hash}`);
+  navigate(`${url.pathname}?${params}${url.hash}`);
 };
 
-export const setURLParam = (param: URLParam, value: string) => {
+export const setURLParam = (param: URLParam, value: string, replace?: boolean) => {
```

Comment:
previously with `useHistory`, we did never replace the url, right? Is it necessary to do this distinction when using `useNavigate`, can't we just have always replace=false?

Reply:
We do in fact
https://github.com/netobserv/network-observability-console-plugin/blob/eadc9beff97a2deef33d0494fc080ed3e31e7612/web/src/utils/url.ts#L44

The replace is usefull to avoid having a huge history at startup, when React hooks set url parameters one by one
https://github.com/netobserv/network-observability-console-plugin/blob/eadc9beff97a2deef33d0494fc080ed3e31e7612/web/src/components/netflow-traffic.tsx#L188-L200

Reply:
Ah, I see